docs/EVE-IMAGE-REPRODUCIBILITY.md

```diff
@@ -0,0 +1,274 @@
+# EVE Reproducibility
+
+A key element of EVE is its consistency. Among other things, that means that it must be possible to rebuild EVE images, and the packages that are composed to form EVE, and get functionally identical bits.
+
+We write "functionally identical" rather than "precisely identical" or just "identical", because the compilation and composition process can change
+elements like timestamps or randomly generated data, leading to minor differences. Ideally, two builds of EVE from the same source would be
+precisely identical; that is a future goal. However, functionally identical images are a requirement for EVE.
```

Comment:
Do we know whether we have such issues today?
It would make sense to try a make pkgs rootfs and see if the shas differ.
If they do it will be hard for a user to verify that something tagged with x.y.z built by lf-edge (or built by anybody else) was really built from the x.y.z source since they can't redo the build on their system and compare the shas.

Reply:
I don't know. I haven't tried it. I also thought about trying it out, but there were different possible scenarios. I would prefer if we can release this document (when ready) with this caveat, and then we can put a targeted effort into either verifying that it is not an issue or fixing it.

Reply:
OK